src/Resources/assets/controllers/accordion_controller.js

```diff
@@ -63,5 +90,19 @@ export default class extends Controller {
 
         return element.parentElement;
     }
+
+
+    /**
+     * @param {HTMLElement|null} element
+     * @param {string} target
+     */
+    closestChildTarget(element, target) {
+        const childElement = element.querySelector(`[data-araise--table-bundle--accordion-target='${target}']`);
```

Comment:
Nit: A more native approach would probably be, using `aria-controls` and matching the elements with IDs, which are either set by userland (devs) or randomly generated (which could be easily done in Twig too).